### DISCUSSION TITLE
Not Opening db

### DISCUSSION THREAD

**Discussion by JeganMurugan** [dbeaver/dbeaver#35357] [Q&A] [answered]
### Description

https://drive.google.com/file/d/1ZEqCKaiRMzTIZFXpJOMljTjmFbkOOHbM/view?usp=sharing  

check this video file 

### DBeaver Version

dbeaver-ce-24.1.5-x86_64

### Operating System

window 11

**Comment by ask9** [comment]
Check error logs in folder:
C:\Users\your_user_name\AppData\Roaming\DBeaverData\workspace6\.metadata

**Comment by ShadelessFox** [comment]
Hello @JeganMurugan,

Please provide log files. Without them, it's impossible to tell what went wrong. 

You can find instructions here: https://github.com/dbeaver/dbeaver/wiki/Log-files#dbeaver-log-files

**Comment by JeganMurugan** [comment]
Here the below i attached the File log i am facing issue unable to open the dbeaver
S.M.Jegan (Developer)

**Comment by ShadelessFox** [comment]
@JeganMurugan you can't attach files when replying by email. Please attach them directly on GitHub.

**Comment by JeganMurugan** [comment]
[metadata.zip](https://github.com/user-attachments/files/16749520/metadata.zip)

**Comment by JeganMurugan** [comment]
@ShadelessFox  i attached the log file check it

**Comment by ShadelessFox** [comment]
It seems that something is wrong with your DBeaver installation. If it was installed using an archive (ZIP), please remove it completely and unpack fresh version instead. https://dbeaver.io/download/

Please let me know if it helps.

**Comment by JeganMurugan** [comment]
@ShadelessFox that also i tried but not  work after install using these setup.exe extension

**Comment by ShadelessFox** [comment]
Did you uninstall DBeaver before installing it?

**Comment by JeganMurugan** [comment]
@ShadelessFox  
Yes

**Comment by ShadelessFox** [comment, marked as answer]
Try the following:
1. Open the explorer
2. In the address line, paste `%APPDATA%\DBeaverData\workspace6\.metadata\.plugins\org.eclipse.core.resources` and hit Enter
3. Remove **everything** from that folder (folders called `.history`, `.projects`, `.root`, and `.safetable`)
4. Try starting DBeaver

**Comment by JeganMurugan** [comment]
@ShadelessFox  now working thank you so much and good support
